services/statestorage/service.go

```diff
@@ -1,44 +1,74 @@
 package statestorage
 
 import (
+	"fmt"
 	"github.com/orbs-network/orbs-network-go/services/statestorage/adapter"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/orbs-network/orbs-spec/types/go/services"
 )
 
 type service struct {
-	services.StateStorage
-	persistence adapter.StatePersistence
+	persistence            adapter.StatePersistence
+	lastResultsBlockHeader *protocol.ResultsBlockHeader
 }
 
 func NewStateStorage(persistence adapter.StatePersistence) services.StateStorage {
 	return &service{
-		persistence: persistence,
+		persistence:            persistence,
+		lastResultsBlockHeader: (&protocol.ResultsBlockHeaderBuilder{}).Build(),
```

Comment:
Hmmm. What did you try to do here?
I can't imagine you'll need to create a state storage with with empty results block

Reply:
Agree, this does not make much sense. 
Service naturally assumes it as at least one block. Since there is no init process yet, this init simulates an ad-hoc genesis block until we implement Init flow.

I think we have two options here - one is to make all the tests pass even when this is nil. The other is to add a //TODO to make sure this is removed once Init flow is implemented. Do you see a third?